tests/testthat/test-check-cardinalities.R

```diff
@@ -48,23 +40,16 @@ test_that("check_cardinality_...() functions are checking the cardinality correc
 
   # expect specific errors and sometimes specific output due to errors ---------------
 
-  pmap(
-    list(
-      d1_src,
-      d2_src,
-      card_0_n_d1_d2_names
-    ),
-    ~ expect_known_output(
-      expect_dm_error(
-        check_cardinality_0_n(
-          parent_table = ..1,
-          pk_column = a,
-          child_table = ..2,
-          fk_column = a
-        ),
-        class = "not_subset_of"
+  verify_output(
```

Comment:
I'd use either `verify_output()` or `expect_error()`, not both.

Reply:
Same reason as above, error plus info-tibble; but yeah, unclear if we need to check both